keps/sig-network/20200525-NodeIPAM-controller-support-variable-sized-CIDRs.md

```diff
@@ -0,0 +1,115 @@
+---
+title: NodeIPAM controller support variable sized CIDRs
```

Comment:
This is annoying, but there is a newer KEP template:  https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template

Now it's a directory with a slightly different structure